docs/README.md

```diff
@@ -0,0 +1,80 @@
+# IBM Cloud Internet Service (CIS)
+IBM Cloud Internet Services, powered by Cloudflare, provides a fast, highly performant, reliable, and secure internet service for customers running their business on IBM Cloud. The CIS provides security, reliabilty and performance features to configure for your domain such as web application firewall (WAF), distributed denial of service (DDoS), global load balancing (GLB) and domain name system(DNS). For more information see, [About IBM Cloud Internet Services](https://cloud.ibm.com/docs/cis?topic=cis-about-ibm-cloud-internet-services-cis).
+
+## Domain
+
+The [domain module](https://github.com/terraform-ibm-modules/terraform-ibm-cis/blob/main/modules/domain/) provides the terraform resources required to add and configure a domain in a CIS instance. After the module run successfully-
```

Comment:
we should say `submodule`  as we mention the same in main readme.
Need to change other places too.